### ISSUE TITLE
Feature request - a button to re-initialize USB port when no device is detected

### ISSUE THREAD

**Issue by arteee87** [ofmodemsandmen/ROOterSource2102#9]
In my case, i use rbm33g with dw5821e modem. After power cycling the router, modem is not detected on USB bus.
Only after physically reconnecting the modem to the USB port it is detected.
RouterOS has an option to toggle USB port power even if there is no device connected.
Would be nice to have a similar thing on rooter.
